inlong-manager/manager-plugins/src/main/java/org/apache/inlong/manager/plugin/util/FlinkConfiguration.java

```diff
@@ -29,7 +29,7 @@
 
 import static org.apache.inlong.manager.plugin.flink.enums.Constants.ADDRESS;
 import static org.apache.inlong.manager.plugin.flink.enums.Constants.JOB_MANAGER_PORT;
-import static org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS;
+import static org.apache.inlong.manager.plugin.flink.enums.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY;
```

Comment:
Why don't you use `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY`?

Reply:
Hi @gong 
I tried adding `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `org.apache.inlong.common.constant.Constants` and also directly using in `inlong-sort` but in both the options were needing to create dependency on 'manager-plugins' which is not correct which might create  circular dependency between modules 'inlong-common' and 'manager-common'.
For reference you can check commit-1

Reply:
> Hi @gong I tried adding `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `org.apache.inlong.common.constant.Constants` and also directly using in `inlong-sort` but in both the options were needing to create dependency on 'manager-plugins' which is not correct which might create circular dependency between modules 'inlong-common' and 'manager-common'. For reference you can check commit-1

@bibhu107 Hi, I think you can `inlong-sort` and `manager-plugin`  module dependency `inlong-common`  after adding `org.apache.inlong.common.constant.Constants.METRICS_AUDIT_PROXY_HOSTS_KEY` to `org.apache.inlong.common.constant.Constants`. `inlong-sort` module don't need to dependency `manager-plugin` module.

Reply:
Hi @gong 

Have considered the above comments and raised a new commit. Please check and let me know if it looks good.

Reply:
> Hi @gong
> 
> Have considered the above comments and raised a new commit. Please check and let me know if it looks good.

look good to me